server/server.go

```diff
@@ -1060,7 +1060,7 @@ func (s *Server) writeID(file string, id uuid.UUID) error {
 func (s *Server) Reload() error {
```

Comment:
If this function can't error anymore we should remove the `error` return value and simplify the caller to not need to handle the error.

Reply:
fixed.